cookbook/tx-new-create-claimable-balance.mdx

```diff
@@ -0,0 +1,258 @@
+---
+title: Create Claimable Balance
+hide_table_of_contents: true
+description:
+  Create claimable balances with various claim predicates using the Stellar CLI
+custom_edit_url: https://github.com/stellar/stellar-cli/edit/main/cookbook/tx-new-create-claimable-balance.mdx
+---
+
+Claimable balances allow you to send payments that can be claimed by the
+recipient based on specific conditions. This is useful for escrow-like
+functionality, conditional payments, or time-locked transfers.
+
+## Setup
+
+For the following examples we will use these accounts:
+
+```sh
```

Comment:
I'll get back to this once Leigh's pr fixing cookbooks is merged (#1963). There are other files that are using `sh` and will need to be changed/fixed.

Reply:
At some point it would be cool to change the way the code blocks work so that the contents are just shoved into a bash script and executed that way. It'd give us greater flexibility to write bash sequences when we want to, and do things like piping too. I left that idea for another time though, because I burnt so much time already on the clean up in #1963.